backend/apps/migrations/management/commands/check_migration_safety.py

```
@@ -0,0 +1,40 @@
+        self.stdout.write('🔍 Checking migration safety...')
+        
+        # Get pending migrations
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT app, name FROM django_migrations")
+            applied = cursor.fetchall()
+        
+        # Analyze migration files
+        issues = []
+        warnings = []
+        
+        for pattern, message in self.DANGEROUS_PATTERNS:
```

Ruff (0.15.21)

[warning] Loop control variable `message` not used within loop body

Rename unused `message` to `_message`

(B007)